### PR TITLE
feat: purge command

### LINKED ISSUE #16: Add Purge Command
Add a purge command for deleting a number of previous messages.

### PR DESCRIPTION
**Pull request by imCHiGi**
Added a command to purge/delete desired number of messages from a channel.
  -Need to have admin permissions.
  -By default it will delete last 5 messages (including the command).

### CODE REVIEW

**Review by Arpan-206 — APPROVED** [review]

**Comment by madebylydia** [comment]
Hey there! Thank you for your contribution for the Discord.py bot of HackSquad!

I'd like to have a few comment about your command:

- Purge should not have a default argument (Actually 5), otherwise, when using `!purge`, 5 message will be purged and this might not be good in case someone just wanted to see the help.
- Your command should preferably have a doctoring explaining the command's purpose.
- Users with the `Manage Messages` should also have access to the command.
- Your command should return a feedback, telling that the purge has been completed or not.

File: hacksquad_discordpy/main.py [hunk 1]
```diff
@@ -31,6 +31,10 @@ async def sync(ctx):
     await bot.tree.sync()
     await ctx.send("Synced")
 
+@bot.command()
```
Comment (Arpan-206) [inline hacksquad_discordpy/main.py:34]: Remove default value and add a Docstring regarding the function

File: hacksquad_discordpy/main.py [hunk 2]
```diff
@@ -31,6 +31,10 @@ async def sync(ctx):
...
+@commands.has_permissions(administrator=True)
+async def purge(ctx, amount=5):
+    await ctx.channel.purge(limit=amount)
```
Comment (Arpan-206) [inline hacksquad_discordpy/main.py:37]: Return a message after successful completion, edits permissions to anyone with `manage messages` permission.

**Review by Arpan-206 — CHANGES_REQUESTED** [review]

**Comment by madebylydia** [comment]
Hey there @imCHiGi, the project has been rewrite from A to Z.
Your changes are probably out of date, do you mind updating them?

It would be nice if you can make a `moderation` cog so that it can easily be worked on more safely!

Thanks for your understanding!

**Comment by madebylydia** [comment]
Closing as there was no update on this.

**Pull request closed without merging**